tests/routers/test_browser_security.py

```
@@ -0,0 +1,78 @@
+    allowed_dir.mkdir()
+
+    # Create MonitoredPath entry
+    path = MonitoredPath(
+        name="Allowed Path",
+        source_path=str(allowed_dir),
+    )
+    db_session.add(path)
+    db_session.commit()
+
+    # Attempt to browse valid path
+    response = viewer_client.get(f"/api/v1/browser/list?path={str(allowed_dir)}")
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)